testslide/mock_callable.py

```diff
@@ -572,10 +572,11 @@ def _registered_calls(self):
 class _MockCallableDSL(object):
 
     CALLABLE_MOCKS: Dict[int, _CallableMock] = {}
+    _NAME: str = "mock_callable"
 
     def _validate_patch(
         self,
-        name="mock_callable",
+        name=_NAME,
```

Comment:
I think we can drop the name argument here, as it can (should!) be accessed with `self._NAME`.